docs/guide/README.md

```diff
@@ -1,32 +1,32 @@
-# 介绍
+# Introduction
 
-dva 首先是一个基于 [redux](https://github.com/reduxjs/redux) 和 [redux-saga](https://github.com/redux-saga/redux-saga) 的数据流方案，然后为了简化开发体验，dva 还额外内置了 [react-router](https://github.com/ReactTraining/react-router) 和 [fetch](https://github.com/github/fetch)，所以也可以理解为一个轻量级的应用框架。
+Dva is first a data flow scheme based on [redux](https://github.com/reduxjs/redux) and [redux-saga](https://github.com/redux-saga/redux-saga). Then to simplify the development experience, dva also has built-in [react-router](https://github.com/ReactTraining/react-router) and [fetch](https://github.com/github/fetch), so can be thought of as a lightweight application framework.
```

Comment:
Dva is ~~first~~ firstly a data flow scheme based on [redux](https://github.com/reduxjs/redux) and [redux-saga](https://github.com/redux-saga/redux-saga). ~~Then to simplify~~ It not only simplifies the development experience, ~~dva~~ also has built-in [react-router](https://github.com/ReactTraining/react-router) and [fetch](https://github.com/github/fetch)~~, so~~. Therefore it can be thought of as a lightweight application framework.